src/game/AI/ScriptDevAI/scripts/outland/hellfire_peninsula.cpp

```diff
@@ -48,69 +48,43 @@ EndContentData */
 
 enum
 {
-    SAY_FREE                        = -1000139,
```

Comment:
I think this entry can be removed from scriptdev2.sql as well, right?

Reply:
You are absolutely correct, i always forget those textes